### DISCUSSION TITLE
编译be失败，求助

### DISCUSSION THREAD

**Discussion by denganming** [apache/doris#10664] [A - General / Q&A]
使用的apache/incubator-doris:build-env-for-1.0.0镜像进入后编译的
[0/2] Re-checking globbed directories...
ninja: error: '/var/local/thirdparty/installed/lib/libkrb5support.a', needed by 'src/service/doris_be', missing and no known rule to make it

**Comment by hf200012** [comment]
你编译的master吧，这个需要使用：docker pull apache/incubator-doris:build-env-ldb-toolchain-latest
